x/auth/src/keeper.rs

```diff
@@ -144,118 +139,65 @@ impl<SK: StoreKey, PSK: ParamsSubspaceKey, M: Module> Keeper<SK, PSK, M> {
         }
     }
 
-    pub fn init_genesis<DB: Database>(
+    pub fn init<DB: Database, CTX: TransactionalContext<DB, SK>>(
```

Comment:
Why `TransactionalContext` vs `InitContext`?

Reply:
I split single method into 2 parts and wanted to make keeper as more generic as possible

Reply:
As you see, I even removed `_genesis` part from name

Reply:
I can't say `yes` or `no`. But probably there wouldn't be such cases

Reply:
> ...make keeper as more generic as possible
I generally aim for the exact opposite :wink:

Reply:
Okay. I revert some changes

Reply:
Edited to use InitContext and removed gas